controllers/reconcilers/kisvc_route_reconciler.go

```diff
@@ -0,0 +1,206 @@
+package reconcilers
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-logr/logr"
+	kservev1beta1 "github.com/kserve/kserve/pkg/apis/serving/v1beta1"
+	"github.com/kserve/kserve/pkg/constants"
+	"github.com/kserve/kserve/pkg/utils"
+	"github.com/opendatahub-io/odh-model-controller/controllers/components"
+	"github.com/opendatahub-io/odh-model-controller/controllers/processors"
+	v1 "github.com/openshift/api/route/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+	"knative.dev/pkg/network"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"strings"
+)
+
+const (
+	IstioNamespace      = "istio-system"
+	IstioIngressService = "istio-ingressgateway"
```

Comment:
Please. create another ticket to properly parametrize this.

Reply:
I had created below ticket to parametrize these env var
https://github.com/opendatahub-io/odh-model-controller/issues/85